pkg/loop/internal/pb/median.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
```

Comment:
Now that we're using protos, do we follow any style/naming guide for proto files?

Will be good to tag onto some existing standards. Otherwise people will keep tripping over personal preferences etc over how to name services, rpcs, fields, etc etc.

Reply:
I'm open to suggestions :shrug: 
I tried to follow the style in the basic guides and GoLand did some formatting for me.

Reply:
I see this: https://protobuf.dev/programming-guides/

Is this the first time we are using protos in Chainlink?
If yes, I'd say lets start with some existing guides.
We could also open a conversation with wider eng group, and get more opinions.
But I feel people will prefer to use some guide.

Reply:
> Is this the first time we are using protos in Chainlink?

No, they are in use in many places already, e.g. libocr telemetry and mercury.